.github/workflows/build-test-distribute.yaml

```diff
@@ -23,9 +20,8 @@ concurrency:
 jobs:
   check:
     permissions:
-      contents: read
-      # golangci-lint-action
-      checks: write
+      checks: write # needed for golangci/golangci-lint-action to add code annotations in PRs
+      contents: write # needed for sca-project to upload sbom to release assets
```

Comment:
We had this `read` and it worked?

Reply:
You are right, it's unnecessary as we are not using this job to publish SBOM/CVE reports to releases - removed